ssl/tls13_enc.c

```diff
@@ -454,7 +456,7 @@ static int derive_secret_key_and_iv(SSL *s, int sending, const EVP_MD *md,
     if (!tls13_derive_key(s, md, secret, key, keylen)
             || !tls13_derive_iv(s, md, secret, iv, ivlen)) {
         /* SSLfatal() already called */
-        goto err;
+        return 0;
```

Comment:
If the second call here fails, won't `key` contain sensitive information which isn't being cleansed?

I guess with the connection dropping, it's not that important.

Reply:
It will be cleaned by the callers of `derive_secret_key_and_iv` in case of any error.

Reply:
Okay, Thanks.  I figured something like this would be the case.